firebase-inappmessaging-display/src/main/java/com/google/firebase/inappmessaging/display/FirebaseInAppMessagingDisplay.java

```diff
@@ -563,4 +563,9 @@ private boolean supportsCustomTabs(Activity activity) {
         activity.getPackageManager().queryIntentServices(customTabIntent, 0);
     return resolveInfos != null && !resolveInfos.isEmpty();
   }
+
+  private boolean shouldOpenWeb(Uri uri) {
```

Comment:
wdyt about having a different name for this method? like ishttpOrHttpsUri or something else since that is what this method is checking.